tests/tests.rs

```diff
@@ -442,9 +442,21 @@ fn one_bit_corruption_quickcheck(tag: u16, data: (Vec<u8>, u8), bit_to_flip: u16
 #[derive(Clone, Debug, PartialEq, Eq, CanonicalSerialize, CanonicalDeserialize)]
 struct Blob(Vec<u8>);
 
+#[tagged("BLOB", compressed)]
+#[derive(Clone, Debug, PartialEq, Eq, CanonicalSerialize, CanonicalDeserialize)]
+struct BlobCompressed(Vec<u8>);
+
+#[tagged("BLOB", checked)]
+#[derive(Clone, Debug, PartialEq, Eq, CanonicalSerialize, CanonicalDeserialize)]
+struct BlobChecked(Vec<u8>);
+
+#[tagged("BLOB", compressed, checked)]
+#[derive(Clone, Debug, PartialEq, Eq, CanonicalSerialize, CanonicalDeserialize)]
+struct BlobCompressedChecked(Vec<u8>);
```

Comment:
my only comment is, we should try applying on types that actually behave differently under compressed and checked flag. meaning `Blob(GroupElement)` instead of wrapping over a vector.

Reply:
One more test added in [8251430](https://github.com/EspressoSystems/tagged-base64/pull/47/commits/82514306405574ab953e86c5d1ada89cba9e70e9)